lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -913,13 +913,22 @@ def combine_multiple_fields(field_prefix: str) -> Optional[List]:
         'social_precautions',
         'no_mask',
         'high_risk_feb2021',
+        'overall_risk_jan2021',
+        'covid_vax',
+        'covid_doses',
+        'vac_name_1',
+        'vac_date',
+        'vac_name_2',
+        'vac_date_2',
```

Comment:
I think `vac_date` and `vac_date_2` should only need to be in `date_questions`, not `string_questions`

Reply:
You're right, thanks for catching that!

Reply:
- [ ] Removed `vac_date` and `vac_date_2` from `string_questions`